VasilzevaAA/lab_1/Main.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main() {
+	std::string in_str; // ������� ������
+	std::cout << "Write a sentance: ";
+	std::getline(std::cin, in_str);
+	in_str = in_str + " "; // ������ ������ ��� ��������� ������� ��������� �� �������
+
+	std::vector <std::string> word; // ������ ��� ���� �� ��������� ����
+	std::vector <int> summ_code; // ������ ��� ���� ����� ��������� ������� word
+
+	int c_word = 0; // ���������� ����

```

Comment:
You don't need separate variable for words count since you can obtain it from summ_code or word e.g. by using word.size()